Parsers/VSISDMI.cs

```diff
@@ -2282,6 +3132,80 @@ public VSISDMI()
             }
         };
 
+        public static BitField MMI_STM_L_VALUE => new BitField
+        {
+            Name = "MMI_STM_L_VALUE",
+            BitFieldType = BitFieldType.UInt16,
+            Length = 16,
+            Comment = "Length of X_VALUE for default value (range: 0-10)."
+        };
+
+        public static BitField MMI_STM_L_VALUE2 => new BitField
+        {
+            Name = "MMI_STM_L_VALUE",
+            BitFieldType = BitFieldType.UInt16,
+            Length = 16,
+            Comment = "Length of X_VALUE for pick-up list value (range: 0-10)."
+        };
+
+        public static BitField MMI_STM_NID_DATA => new BitField
+        {
+            Name = "MMI_STM_NID_DATA",
+            BitFieldType = BitFieldType.UInt8,
+            Length = 8,
+            Comment = "Identifier of a Specific STM Data to be entered."
+        };
+
+        public static BitField MMI_STM_Q_FOLLOWING => new BitField
+        {
+            Name = "MMI_STM_Q_FOLLOWING",
+            BitFieldType = BitFieldType.Bool,
+            Length = 1,
+            Comment = "Indicate following data to be viewed at the same time"
+        };
+
+        // TODO Johan, please check whether the length is required along with the scaling factor
+        public static BitField MMI_STM_X_CAPTION => new BitField
+        {
+            Name = "MMI_STM_X_CAPTION",
+            BitFieldType = BitFieldType.StringLatin,
+            Length = 8,
+            Comment = "Data label caption text byte string",
+            VariableLengthSettings = new VariableLengthSettings
+            {
+                Name = "MMI_L_CAPTION",
+                ScalingFactor = 8
+            }
+        };
+
+        // TODO Johan, please check whether the length is required along with the scaling factor
+        public static BitField MMI_STM_X_VALUE => new BitField
+        {
+            Name = "MMI_STM_X_VALUE",
+            BitFieldType = BitFieldType.StringLatin,
+            Length = 8,
+            Comment = "Data value caption text byte string",
+            VariableLengthSettings = new VariableLengthSettings
+            {
+                Name = "MMI_STM_L_VALUE",
+                ScalingFactor = 8
+            }
+        };
+
+        // TODO Johan, please check whether the length is required along with the scaling factor
+        public static BitField MMI_STM_X_VALUE2 => new BitField
```

Comment:
This one looks suspiciously similar to the previous one

Reply:
Yes, they are the same, but VSIS refers to them as different variables. I am happy to reuse MMI_STM_X_VALUE, but just keep in mind that it won't exactly align with the VSIS structure.